src/supporting-data/taxonomy/config/TaxonomyColumnConfiguration.tsx

```diff
@@ -178,6 +178,18 @@ TaxonomyColumnConfiguration.set(TaxonomyColumn.statistics, {
   ),
 });
 
+TaxonomyColumnConfiguration.set(TaxonomyColumn.links, {
+  label: 'See also',
+  render: ({ links }) =>
+    links && (
+      <ExpandableList displayNumberOfHiddenItems descriptionString="links">
+        {links.map((link) => (
+          <ExternalLink url={link}>{link}</ExternalLink>
+        ))}
+      </ExpandableList>
+    ),
+});
+
```

Comment:
This seems to duplicate the existing `TaxonomyColumn.links` column config. The only difference I see is the existing column has the label of `Links`.

Reply:
Not sure how I missed that!